common/app/conf/audio/FlagshipFrontContainer.scala

```diff
@@ -0,0 +1,41 @@
+package conf.audio
+
+import org.joda.time.DateTime
+import org.joda.time.DateTimeConstants._
+import org.joda.time.format.DateTimeFormat
+import scala.concurrent.duration._
+import conf.switches.Switches.FlagshipFrontContainerSwitch
+
+object FlagshipFrontContainer {
+  private val FrontContainerIds = Seq(
+    "75ef80cd-2f3d-40d6-abf6-2021f88ece8e", //PROD
+    "c57a70c8-a00a-4a15-93a2-035b9221622b"  //CODE
+  )
+
+  private val GoLiveDateTime = DateTimeFormat.forPattern("yyyy/MM/dd HH:mm").parseDateTime(s"2018/11/01 03:15")
+
+  //The container should appear at 3:15 on Monday, and disappear at 3:15 on Saturday
```

Comment:
Hmm, question: "3:15 on Saturday" should this be reflected in the email container PR?

Reply:
The requirements were different for the email

Reply:
(and now they're both the same!)